src/JwtGuard/JwtTokenAuthenticator.php

```diff
@@ -150,4 +151,12 @@ public function getSegments()
         $segments = $this->getClaimOrNull( self::SEGMENTS );
         return is_array( $segments ) ? $segments : [ ];
     }
+
+    private function extractJwtFromHeader( $header )
+    {
+        if( strpos( $header, self::BEARER ) !== false ){
```

Comment:
``` strpos( $header, self::BEARER ) === 0``` Seems safer to me, for example if the token contains the ```self::BEARER``` string (should be impossible in its current state), but may not always be true in future.